server/src/main/java/org/opensearch/action/admin/cluster/remotestore/package-info.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/** Restore remote store transport handler. */
+package org.opensearch.action.admin.cluster.remotestore;
```

Comment:
This is not the right package (inside action) for repo registration for remote node. One suggestion would be org.opensearch.node.remotestore